equinox-shipping/Domain/Container.fs

```diff
@@ -0,0 +1,51 @@
+module Shipping.Domain.Container
+
+let [<Literal>] private Category = "Container"
+let streamName (containerId : ContainerId) = FsCodec.StreamName.create Category (ContainerId.toString containerId)
+
+// NB - these types and the union case names reflect the actual storage formats and hence need to be versioned with care
+module Events =
+
+    type Event =
+        | Finalized
```

Comment:
@Kimserey we _could_ make this event bear the Assigned shipmentIds in order to exemplify that the process manager can shuttle state across aggregates ...

IRL, does the list of proposed shipmentIds accumulate on the Aggregate, or is it something that's managed by some external agent / UI ?